harness/determined/cli/experiment.py

```diff
@@ -195,6 +195,21 @@ def create(args: Namespace) -> None:
         submit_experiment(args)
 
 
+def limit_offset_paginator(
+    method: Callable, agg_field: str, connection_args: Namespace, **kwargs: Any
+) -> List[Union[bindings.v1Experiment, bindings.trialv1Trial]]:
```

Comment:
can it just be a `List`? we may have many paged API types, and explicitly specifying it here would be a chore.

Reply:
ok I figured out what I want to do is make this List[Any] and use the places where we call this function to expect a typed list